javascripts/govuk/multivariate-test.js

```diff
@@ -75,14 +75,23 @@
   };
 
   MultivariateTest.prototype.setCustomVar = function(cohort) {
-    if (this.customDimensionIndex) {
-      GOVUK.analytics.setDimension(
-        this.customDimensionIndex,
-        this.cookieName() + "__" + cohort
-      );
+    if (this.customDimensionIndex && Array.isArray(this.customDimensionIndex)) {
+      var _this = this;
+      this.customDimensionIndex.forEach(function(dimensionIndex) {
+        _this.setDimension(cohort, dimensionIndex)
+      });
```

Comment:
Could consider using `function(){}.bind(this)` to give this closure the right context.

Then you don't need to assign a `_this` variable.